crates/matrix-sdk-crypto/src/store/crypto_store_wrapper.rs

```diff
@@ -175,14 +175,22 @@ impl CryptoStoreWrapper {
             if let Some(own_identity_after) = maybe_own_identity.as_ref() {
                 // Only do this if our identity is passing from not verified to verified,
                 // the previously_verified can only change in that case.
-                if !own_identity_was_verified_before_change && own_identity_after.is_verified() {
+                let own_identity_after_is_verified = own_identity_after.is_verified();
+                if !own_identity_was_verified_before_change && own_identity_after_is_verified {
                     debug!("Own identity is now verified, check all known identities for verification status changes");
                     // We need to review all the other identities to see if they are verified now
                     // and mark them as such
                     self.check_all_identities_and_update_was_previously_verified_flag_if_needed(
                         own_identity_after,
                     )
                     .await?;
+                } else if own_identity_was_verified_before_change != own_identity_after_is_verified
+                {
+                    // just log
+                    debug!(
+                        "Own identity trust has changed: isVerified: {}",
```

Comment:
`isVerified` is not really a thing we have in the Rust codebase,  we should probably reword this sentence completely. We should probably avoid using method names, no matter if they are from the FFI layer or the Rust codebase, if the method gets renamed this will be forgotten.